modules/mono/glue/Managed/Files/Colors.cs

```diff
@@ -0,0 +1,303 @@
+using System;
+using System.Collections.Generic;
+
+namespace Godot
+{
+    public static class Colors
+    {
+        // Color names and values are derived from core/color_names.inc
+        internal static readonly Dictionary<string, Color> namedColors = new Dictionary<string, Color> {
+            {"aliceblue", new Color(0.94f, 0.97f, 1.00f)},
+            {"antiqueWhite", new Color(0.98f, 0.92f, 0.84f)},
```

Comment:
antiqueWhite name is uppercase White, does not match the lookup below or all the other lower case names

Reply:
Nice catch.